.travis.yml

```diff
@@ -20,6 +20,9 @@ script:
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_not_implemented.yaml -l multiple_facts.not_implemented -e "ignore_notimplemented=true"
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_not_implemented.yaml -l multiple_facts.not_implemented -e "ignore_notimplemented=false"
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_error.yaml -l multiple_facts.error
+    - ansible-playbook -i napalm_cli/hosts -l multiple_commands.ok napalm_cli/multiple_commands.yaml
+    - ansible-playbook -i napalm_cli/hosts -l wrong_commands.err napalm_cli/wrong_args.yaml
+    - ansible-playbook -i napalm_cli/hosts -l multiple_commands.ok napalm_cli/check_mode.yaml -C
```

Comment:
Can we add these to run_tests.sh (or have .travis.yml use run_tests.sh)? Basically, it is nice to have an easy way to manually run the tests.